### PR TITLE
Feature/markie/dp 10242  checkbox

### PR DESCRIPTION
**Pull request by mark-casias**
## Description
Copies logic of checkbox input from patternlab to react.

## Steps to Test
[ ] - Checkout branch
[ ] - npm run start
[ ] - Navigate to http://localhost:6006/?selectedKind=atoms%2Fforms&selectedStory=InputCheckBox&full=0&addons=1&stories=1&panelRight=0&addonPanel=storybook%2Factions%2Factions-panel
[ ] - Observe checkbox item

## Screenshots
Use something like [licecap](http://www.cockos.com/licecap/) to capture gifs to demonstrate behaviors.
image

#### Today I learned...
Checkboxes need to set state on change otherwise they are not interactive in react.

### CODE REVIEW

File: react/src/components/atoms/forms/InputCheckBox/InputCheckBox.stories.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+
+import { storiesOf } from '@storybook/react';
+import { withInfo } from '@storybook/addon-info';
+import SvgWheelchair from '../../icons/SvgWheelchair';
```
Comment (pmallett-mc) [inline react/src/components/atoms/forms/InputCheckBox/InputCheckBox.stories.js]: I think the needs to be changed to use Icon now -- `npm start` is failing with:

> ERROR in ./src/components/atoms/forms/InputCheckBox/InputCheckBox.stories.js
Module not found: Error: Can't resolve '../../icons/SvgWheelchair' in '/Users/petermallett/workspace/mass_gov/mayflower/react/src/components/atoms/forms/InputCheckBox'
 @ ./src/components/atoms/forms/InputCheckBox/InputCheckBox.stories.js 5:0-54

File: react/src/components/atoms/forms/InputCheckBox/index.js [hunk 2]
```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+// import './style.css';
```
Comment (pmallett-mc) [inline react/src/components/atoms/forms/InputCheckBox/index.js]: I think we may still need this import for now? @clairesunstudio, should this be uncommented, or just be deleted?

**Review by pmallett-mc — CHANGES_REQUESTED** [review]
'couple minor issues to work out now that Icon is in

File: react/src/components/atoms/forms/InputCheckBox/InputCheckBox.stories.js (hunk 1 above)
Comment (mark-casias) [reply react/src/components/atoms/forms/InputCheckBox/InputCheckBox.stories.js]: Just updated this. The icon component was not available when this started.

**Review by pmallett-mc — APPROVED** [review]

**Comment by pmallett-mc** [comment]
Approved w/ minor question/comment for @clairesunstudio about style.css

**Pull request closed without merging**